src/actions/guides/database/database_setup.cr

```diff
@@ -49,6 +49,55 @@ class Guides::Database::DatabaseSetup < GuideAction
     This causes Lucky to raise an exception if you forget to preload an association,
     but will not raise an exception in production.
 
+    ## Multiple Databases
+
+    Avram supports a multi-database setup which you may need to use for connecting to a legacy
+    db, or maybe doing a read/write replica setup.
+
+    By default, Lucky gives you the `AppDatabase` class for your primary DB. To add a second one,
+    you'll need to create a new class and inherit from `Avram::Database`.
+
+    ```crystal
+    # config/database.cr
+    class SecondaryDatabase < Avram::Database
+    end
+    ```
+
+    Next, you'll need to add the connection info for the `SecondaryDatabase`.
+
+    ```crystal
+    # config/database.cr
+    SecondaryDatabase.configure do |settings|
+      settings.url = ENV["SECOND_DATABASE_URL"]? || Avram::PostgresURL.build(
+        database: "db_two",
+        hostname: "localhost",
+        username: "postgres",
+        password: "postgres"
+      )
+    end
+    ```
+
+    Lastly, any models that need to use this database will need to inherit from a new
+    base model class that specifies the `SecondaryDatabase`
```

Comment:
This is not *quite* right. You can do this without making a new abstract class. You can override the database in the model itself with `def  self.database`. I think that's a better option to start so you can share stuff between all models. 

Maybe the "extract abstract class" could be in another `###` section as another possibility for refactoring?